WalletWasabi.Fluent/Views/CoinControl/SelectCoinsDialogView.axaml

```diff
@@ -0,0 +1,119 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:converters="clr-namespace:WalletWasabi.Fluent.Converters"
+             xmlns:generic="clr-namespace:System.Collections.Generic;assembly=System.Collections"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="650"
+             xmlns:coinControl="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl"
+             x:DataType="coinControl:SelectCoinsDialogViewModel"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.SelectCoinsDialogView">
+
+  <controls:ContentArea
+    EnableBack="{Binding EnableBack}"
+    EnableNext="True"
+    NextContent="Continue"
+    EnableCancel="{Binding EnableCancel}"
+    IsBusy="{Binding IsBusy}"
+    ScrollViewer.VerticalScrollBarVisibility="Disabled">
+
+    <controls:ContentArea.Styles>
+      <Style Selector="TreeDataGridRow">
+        <Setter Property="Focusable" Value="False" />
```

Comment:
I think disabling selection is not best idea, in con list we have similar coin selection concept and you can select rows, for me it feels unnatural ? What is reason for that ?

Reply:
You're right. Some conflicting styles were defined. Fixed :)